bodhi/client/__init__.py

```diff
@@ -41,6 +41,19 @@
                  is_flag=True, default=False)]
 
 
+# Common options for the overrides save and edit command
+save_edit_options = [
+    click.argument('nvr'),
+    click.option('--duration', default=7, type=click.INT,
+                 help='Number of days the override should exist.'),
+    click.option('--notes', default="No explanation given...",
+                 help='Notes on why this override is in place.'),
+    click.option('--user', envvar='USERNAME'),
+    click.option('--password', prompt=True, hide_input=True),
+    click.option('--staging', help='Use the staging bodhi instance',
+                 is_flag=True, default=False)]
```

Comment:
If you add ```url_option``` to this list, you won't need to include ```@url_option``` on both the save and edit commands.